clients/src/main/java/org/apache/kafka/clients/admin/MemberDescription.java

```diff
@@ -131,13 +140,30 @@ public Optional<MemberAssignment> targetAssignment() {
         return targetAssignment;
     }
 
+    /**
+     * The epoch of the group member.
+     */
+    public Optional<Integer> memberEpoch() {
+        return memberEpoch;
+    }
+
+    /**
+     * The flag indicating whether a member is classic.
+     */
+    public Optional<Boolean> isClassic() {
```

Comment:
Sorry for coming back on this one but I just thought about an alternative. How about calling it `nonUpgraded`? We could set it to true when the group is a consumer group and the member uses classic protocol. We could also only set it when the group is a consumer group vs a classic group. This may be even more explicit for users. What do you think? @chia7712 Do you have any thoughts too?

Reply:
>  How about calling it nonUpgraded? We could set it to true when the group is a consumer group and the member uses classic protocol. 

How about using "upgraded"? it is true only if the group is a consumer group and the member uses the consumer protocol.

Reply:
That works too. Something as follow?
* Classic Group -> Optional.empty
* Consumer Group -> Optional.empty if unknown
* Consumer Group -> Optional.of(false) if classic
* Consumer Group -> Optional.of(true) if consumer

Reply:
Thanks for the suggestion. Updated PR and KIP.